### DISCUSSION TITLE
Date format does not seem to be respected in filter query strings

### DISCUSSION THREAD

**Discussion by boris-glumpler** [filamentphp/filament#4692] [Help]
Hello,

while not really important, it seems that the format of a `DatePicker` does not get respected in the query string.

In my particular case the date pickers are configured like so:

```php
DatePicker::make('until')
          ->label('until')
          ->default(now())
          ->format('Y-m-d')
```

Here's a screenshot of the filter:

![created_at-filter](https://user-images.githubusercontent.com/580004/197393667-c562e786-8ca6-407f-90f5-4b948d343a48.png)

This results in a query string like this, with the time still there:

```
/admin/pages?tableFilters[created_at][until]=2022-10-23+12%3A51%3A52
```

Is there a way to remove the time? It'd just look a bit cleaner that way.

**Comment by henninghorn** [comment]
I have also noticed this.

In my case, I can't deep link into a form that's using Filament because of this behaviour, it seems.

I have couple of date fields (`DatePicker` to be exact), and they are live and are updating the query string parameters.
However, they also store the time in these query parameters, and the field only accepts strings WITH the additional time, instead of just accepting "Y-m-d", as specified in format("Y-m-d") call when setting up the fields.
